positron/components/Process.js

```diff
@@ -40,6 +40,21 @@ Process.prototype = {
   init: function(window) {
     this._contentWindow = window;
 
+    // The binding appears to get initialized even for a document in a content
```

Comment:
Now that we have a system principal can't we get rid of this and specify `JavaScript-global-privileged-property` in the manifest instead?